ops/services/virtual_network/main.tf

```diff
@@ -0,0 +1,46 @@
+# Modularized version of our standard virtual network configuration
+# To migrate to this from the hard-coded version, you will need to run the
+# following commands from the 'persistent' directory:
+# terraform state mv azurerm_virtual_network.vn module.vnet.azurerm_virtual_network.vn
+# terraform state mv azurerm_subnet.vms module.vnet.azurerm_subnet.vms
+# terraform state mv azurerm_subnet.lbs module.vnet.azurerm_subnet.lbs
+# terraform state mv azurerm_subnet.webapp module.vnet.azurerm_subnet.webapp
```

Comment:
Thanks for capturing these.

Reply:
When would we need to do this - before deploying, I'm assuming, to avoid destructive actions in the pipeline?

Reply:
Before deploying this module, which is actually only deployed sporadically and by hand. 😕